src/functions/slack-user-avatar/handler.ts

```diff
@@ -7,10 +7,43 @@ import TimeBankApiProvider from "src/meta-assistant/timebank/timebank-api";
  * Response schema for lambda
  */
 export interface Response {
-  personId: number,
-  image_original: string
+  personId: number;
+  image_original: string;
 }
 
+/**
+ * Cache for Slack users data
+ */
+let slackUsersCache: { users: any[], expiresAt: number } | null = null;
```

Comment:
Add some logging to make debugging/ testing easier. e.g. some simple log here to show if slack user cache is null or not